admin/organization/security-settings/sso-configuration.md

```diff
@@ -49,16 +54,16 @@ Follow the steps on this page to configure SSO for your organization.
 
    ![Azure AD](../../../docker-hub/images/azure-create-connection.png){: width="500px" }
 
-4. From your IdP, copy and paste the following values into the Docker **Settings** fields:
+7. From your IdP, copy and paste the following values into the Docker **Settings** fields:
```

Comment:
Same as [this comment](https://github.com/docker/docs/pull/17705/files#r1256325405)